dance/transforms/normalize.py

```diff
@@ -575,9 +577,10 @@ class NormalizeTotal(AnnDataTransform):
 
     """
 
-    def __init__(self, exclude_highly_expressed_and_max_fraction: Optional[float] = None, **kwargs):
-        if exclude_highly_expressed_and_max_fraction < 0:
-            super().__init__(sc.pp.normalize_total, exclude_highly_expressed=False, **kwargs)
-        else:
-            super().__init__(sc.pp.normalize_total, exclude_highly_expressed=True,
-                             max_fraction=exclude_highly_expressed_and_max_fraction, **kwargs)
+    def __init__(self, max_fraction: Optional[float] = None, **kwargs):
+
+        super().__init__(sc.pp.normalize_total, exclude_highly_expressed=True, max_fraction=max_fraction, **kwargs)
```

Comment:
Do not super init the funciton to be wrapped (same for *all* other transformation you are going to wrap). You should only super init and inherit from the base transform function. Other wise it won't be compatible with the pipeline.